source/docs/01-introduction.xml

```diff
@@ -739,22 +739,22 @@
                         that notation are specified in chapter <ptr target="#mensural"/>, even though
                         other chapters of these Guidelines apply as well.</item>
                      <label>mei-Neumes</label>
```

Comment:
```suggestion
                     <label>mei-neumes</label>
```

Reply:
@rettinghaus Plus `mei-cmn` in line 730 and `mei-mensural` in line 736, but I cannot reach those lines with the review.